src/parallelzone/runtime/detail_/runtime_view_pimpl.hpp

```diff
@@ -158,6 +172,9 @@ struct RuntimeViewPIMPL {
      *  ResourceSet in a const function we can do that.
      */
     mutable resource_set_container m_resource_sets_;
+
+    /// Stacks of initialize and finalize callback functions
+    std::stack<callback_function_type> m_callbacks_final;
```

Comment:
```suggestion
    std::stack<callback_function_type> m_callbacks_final_;
```
This is a private member so it should end in an underscore. You're going to have to work the new name through the file.